README.md

```diff
@@ -1,5 +1,16 @@
 # becquerel
```

Comment:
Maybe we should also point out that to install becquerel it is best to use the `--user` flag, which puts it into a user owned directory, compared to the way described below (which often requires root access). It might even make sense to add this as the default way of installing it (not sure if it is cross platform compatible). At least for newcomers that might be very valuable information.

Reply:
Good point.

Reply:
This change was made with 3f416cf (comment remains visible since another line was changed).